tests/tests_geomstats/test_spd_matrices.py

```diff
@@ -19,192 +22,156 @@
     SPDMetricEuclidean,
     SPDMetricLogEuclidean,
 )
+from tests.conftest import Parametrizer, generate_tests
 
 SQRT_2 = math.sqrt(2)
 
 
-class TestSPDMatrices(geomstats.tests.TestCase):
-    """Test of SPDMatrices methods."""
-
-    def setup_method(self):
-        """Set up the test."""
-        warnings.simplefilter("ignore", category=ImportWarning)
+def belongs_data():
+    smoke_data = [
+        dict(n=2, mat=[[3.0, -1.0], [-1.0, 3.0]], expected=True),
```

Comment:
Any reason why you use the `dict` syntax as opposed to `{}` with keys as strings? (no need to change everything, I am just curious).